openlattice.yaml

```diff
@@ -2912,6 +2912,113 @@ paths:
             application/json:
               schema:
                 type: integer
+  /datastore/admin/reload/cache:
+    get:
+      summary: Reload all the in memory caches
+      operationId: reloadCache
+      tags:
+        - admin
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      responses:
+        "200":
+          description: Success
+  /datastore/admin/reload/cache/{name}:
+    get:
+      summary: Reload the in memory caches for the specified name
+      operationId: reloadSpecificCache
+      tags:
+        - admin
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      parameters:
+        - name: name
+          in: path
+          required: true
+          schema:
+            type: string
+            enum: [
+              ACL_KEYS,
```

Comment:
some of these are in-memory maps only and can't be reloaded (this will only work on ones backed by a mapstore)

Reply:
where can I find which ones?

Reply:
Not easily... but all maps that have mapstores are in MapstoresPod class.